src/js/components/ColumnChooser.js

```diff
@@ -13,24 +12,15 @@ type Props = {}
 export default class ColumnChooser extends React.Component<Props> {
   render() {
     return (
-      <div className="columns-button">
+      <div className="columns-button" title="Show or hide columns in the table">
```

Comment:
In the spirit of "as native as possible", I think we should use the standard html title attributes. They look great, they always get positioned properly, and it appears they have a global timer for all of them, making the user not have to wait the full delay if they jump from one button to another.

<img width="319" alt="Screen Shot 2020-05-20 at 5 18 34 PM" src="https://user-images.githubusercontent.com/3460638/82510005-53e1eb80-9abe-11ea-8c8b-6e97a349cecd.png">

Reply:
Agreed. Simple and functional is very good.